static-build/CMakeLists.txt

```diff
@@ -211,7 +211,7 @@ if (APPLE)
     # So we need to find library and header and
     # copy it locally
     find_path(UNWIND_INCLUDE_DIR libunwind.h)
-    find_library(UNWIND_LIBRARY libunwind.dylib
+    find_library(UNWIND_LIBRARY libunwind.tbd
```

Comment:
Shouldn't we do it conditionally?

OTOH, we dropped Mac OS 10.15 support recently: 280267227b1e0f9a1694aae574547e4565a6d6b0. So, maybe, the answer is 'no'.

Reply:
I also think condition are not necessary by this reason.